holoviews/plotting/bokeh/raster.py

```diff
@@ -197,8 +197,8 @@ def get_data(self, element, ranges, style):
             XS, YS = [], []
             for x, y, zval in zip(X, Y, zvals):
                 if np.isfinite(zval):
-                    XS.append(x)
-                    YS.append(y)
+                    XS.append(list(x[:-1]))
+                    YS.append(list(y[:-1]))
```

Comment:
I assume there is a good reason though it seems odd to me you are slicing off the last element...

Reply:
When drawing a polygon it automatically connects the first and last point, so the last point for each quad geometry is superfluous.

Reply:
Ok, that does make sense. Thanks!